SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialWorkerFlags/WorkerFlagsTestSpyObject.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "WorkerFlagsTestSpyObject.h"
+
+void  UWorkerFlagsTestSpyObject::SetFlagUpdated(const FString& flagName, const FString& flagValue)
+{
+	timesUpdated++;
```

Comment:
All our vars should be UpperCamelCase :D
Could you sort this for all the classes.